cmd/clusterctl/client/cluster/mover.go

```diff
@@ -333,6 +335,19 @@ func (o *objectMover) move(graph *objectGraph, toProxy Proxy, mutators ...Resour
 		return errors.Wrap(err, "error pausing ClusterClasses")
 	}
 
+	log.Info("Waiting for all resources to be ready to move")
+	// exponential backoff configuration which returns durations for a total time of ~2m.
+	// Example: 0, 5s, 8s, 11s, 17s, 26s, 38s, 57s, 86s, 128s
```

Comment:
Just curious, are these the actual numbers it would use or did you pull this from somewhere?

Reply:
The jitter makes this inherently random to some degree, but I threw together something to simulate what these numbers could be and this is one example.